apps/web/next.config.js

```diff
@@ -47,7 +47,7 @@ module.exports = withImages(
       ];
 
       config.module.rules.push({
-        test: /\.(woff|woff2|eot|ttf|otf)$/,
+        test: /\.woff|woff2|eot|ttf|otf$/,
```

Comment:
I think that it's a bit more readable with parens than without. But I don't have a strong objection.

Reply:
@tbergq Should be `/\.(?:woff|woff2|eot|ttf|otf)$/` otherwise the first dot is only for `woff` which is different from the previous RegExp behavior. Not a big deal in a real life though.

Reply:
thank you @mrtnzlml I will fix it

Reply:
@mrtnzlml 🚀